cmd/skopeo/utils_test.go

```diff
@@ -54,6 +54,7 @@ func TestImageOptionsNewSystemContext(t *testing.T) {
 		"--dest-daemon-host", "daemon-host.example.com",
 		"--dest-tls-verify=false",
 		"--dest-creds", "creds-user:creds-password",
+		"--dest-registry-token", "faketoken",
```

Comment:
`--src-registry-token` tests too?

Reply:
As commented below, testing `--src` is not necessary, as we are just doing the test with an arbitrary prefix.